ui/analyse/css/explorer/_config.scss

```diff
@@ -117,10 +117,26 @@
     }
     .previous {
       @extend %flex-wrap;
+      grid-template-columns: auto auto;
+      column-gap: 0.5em;
       .button {
         margin: 0.3em 0.2em;
         text-transform: none;
       }
+      .remove {
+        margin-left: -1.75em;
+        width: 1.75em;
+        height: 1.75em;
+        top: -1.5em;
+        right: -0.65em;
```

Comment:
```css
       margin-left: -1.75em;
        width: 1.75em;
        height: 1.75em;
        top: -1.5em;
        right: -0.65em;
```
that looks extremely fragile :-/ is there a better way that doesn't require 5 constants to maintain?